app/index/controller.js

```diff
@@ -75,20 +72,28 @@ export default Controller.extend({
       "ymax": bbox[1][1],
     };
     let extent = `${bb.xmin},${bb.ymin},${bb.xmax},${bb.ymax}`;
-    let searchString = `${geocodeUrl}suggest?text=${text}&searchExtent=${extent}&f=json`;
-    return this.get('ajax').request(searchString)
+
+    // construct the search string
+    let searchString = `${geocodeUrl}suggest?text=${text}&f=json`;
+    // if we aren't purposefully removing the extent, then add it into the search string
+    let removeExtentFromQuery = this.get('removeExtentFromQuery');
+    if (!removeExtentFromQuery) searchString = `${searchString}&searchExtent=${extent}`;
+    debug(`Sending /suggest query with searchString: ${searchString}`);
+    return this.get('ajax').request(searchString);
   },
 
   _setAddress (address) {
```

Comment:
I'd consider renaming this, as it's not *just* setting the address - it's actually getting the address candidates...